src/Language/PureScript/Errors.hs

```diff
@@ -781,6 +771,14 @@ prettyPrintSingleError full level e = do
       paras [ line $ "The import of type " ++ runModuleName name ++ " as " ++ runModuleName qualName ++ " uses the deprecated 'import qualified' syntax."
             , line $ "This syntax form will be removed in PureScript 0.9." ]
 
+    renderSimpleErrorMessage (DeprecatedClassImport mn name) =
+      paras [ line $ "The import of class " ++ runProperName name ++ " from " ++ runModuleName mn ++ " uses the deprecated syntax that omits the class keyword."
```

Comment:
We'd have to reconstruct the whole import/export declaration to be able to do that in a useful manner I think - what can we show them apart from `class SomeProperName`?